aicodebot/lm.py

```diff
@@ -235,17 +224,20 @@ def read_model_config(self):
             key_name = "OPENAI_API_KEY"
         elif self.provider == self.OPENROUTER:
             key_name = "OPENROUTER_API_KEY"
-        elif self.provider == self.HUGGINGFACE_HUB:
-            key_name = "HUGGINGFACE_API_KEY"
+        elif self.provider == self.OLLAMA:
+            key_name = "OLLAMA_LOCAL"
         else:
             raise ValueError(f"Unrecognized provider: {self.provider}")
-
-        if not self.get_api_key(key_name):
+        model = self.get_key(key_name)
```

Comment:
I don't agree with this change. 

`get_api_key` is more explicit and clear.

Revert or push back if there is a good reason.